test/pjrt/test_runtime.py

```diff
@@ -55,14 +55,12 @@ def test_xla_device_error(self):
   }, False), ('pjrt_cpu', {
       'PJRT_DEVICE': 'CPU',
       'PJRT_SELECT_DEFAULT_DEVICE': '0'
-  }, True), ('xrt_tpu', {
-      'XRT_TPU_CONFIG': 'localservice;0;localhost:51011'
-  }, False), ('pjrt_tpu_precedence', {
+  }, True), ('pjrt_tpu_precedence', {
       'PJRT_DEVICE': 'TPU',
       'XRT_TPU_CONFIG': 'localservice;0;localhost:51011',
-  }, True), ('xrt_gpu', {
+  }, True), ('gpu_num_devives', {
       'GPU_NUM_DEVICES': '4'
-  }, False), ('pjrt_gpu', {
+  }, True), ('pjrt_gpu', {
       'PJRT_DEVICE': 'GPU',
       'GPU_NUM_DEVICES': '4'
   }, True), ('xla_dist_worker', {
```

Comment:
do we still want to keep this 'xla_dist_worker' test? not sure what this test for and wanna discuss here

Reply:
Good catch. I missed this on my first pass.

Reply:
This tested that the PJRT device wouldn't be set if you started XRT distributed training with `xla_dist`. This PR removes `xla_dist`, so we can remove this case.

Reply:
Thanks for clarification, got it!